sdk/storage/ci.yml

```diff
@@ -22,6 +22,13 @@ pr:
     - sdk/storage/
     - sdk/eng/
 
+resources:
+  repositories:
+  - repository: SwiftPM-AzureStorageBlob
```

Comment:
I don't think these exist yet since these aren't even in preview release, in case that matters.

Reply:
I'll remove them and put a skip in for the SPM package publish job.